app/main.py

```diff
@@ -371,32 +399,179 @@ def update_live_alerts_data(n_intervals, ongoing_live_alerts, ongoing_frame_urls
+                    try:
+                        # For each new live alert, we fetch the URL of the associated frame
+                        img_url = api_client.get_media_url(row["media_id"]).json()["url"]
+
+                    except Exception:
+                        # This is just a security in case we cannot retrieve the URL of the detection frame
+                        img_url = ''
+
+                    # We update the detection frame URL dictionary with the same method as above
+                    if str(row['event_id']) not in dict_images_url_live_alerts.keys():
+                        dict_images_url_live_alerts[str(row['event_id'])] = [img_url]
+
+                    else:
+                        dict_images_url_live_alerts[str(row['event_id'])].append(img_url)
+
+                # Is there any new event among these new alerts?
+                condition = (~new_alerts['event_id'].isin(ongoing_live_alerts['event_id'].unique())).sum()
+
+                # If this condition is verified, this means that there is a new "alert" (in fact an event) to display
+                # on the platform and we therefore need to update all components (the live_alert_header_btn, the user
+                # selection area, etc)
+                if condition:
+
+                    # Merging yaw (azimuth) field from devices_data
+                    all_devices = pd.DataFrame(devices_data)
+
+                    # We follow the same process as above to replace the azimuth of the alert with the yaw of the device
+                    devices_yaw = all_devices[['id', 'yaw']].copy()
 
-    return live_alerts, dict_images_url_live_alerts
+                    live_alerts = pd.merge(
+                        live_alerts, devices_yaw,
+                        how='left',
+                        left_on=['device_id'], right_on=['id']
+                    )
+
+                    live_alerts = live_alerts.drop(['azimuth'], axis=1)
+
+                    live_alerts.rename(columns={'id_x': 'id', 'id_y': 'd_id'}, inplace=True)
+
+                    live_alerts = live_alerts.to_json(orient='records')
+
+                    # We update all outputs
+                    return [
+                        live_alerts,
+                        dict_images_url_live_alerts,
+                        {'loaded_frames': new_loaded_frames},
+                        dash.no_update
+                    ]
+
+                # If the condition is not verified, we have no new "alert" / event to display on the platform but only
+                # new detection frames for an existing alert; this means that we do not have to update all components
+                else:
+
+                    # We would like to only update the list of alert frames being displayed and not all the components
+                    # To keep track of the frame URLs that have been loaded, we also update the list of loaded alert IDs
+                    return [
+                        dash.no_update,
```

Comment:
Nice dash.no_update for a specific output, still impressed of endless dash framework deepness 😅